mk/kafka.mk

```
@@ -98,16 +22,16 @@ kafka-topics-list:  ## List the kafka topics from the kafka container
 
 .PHONY: kafka-topics-create
 kafka-topics-create:  ## Create the kafka topics in KAFKA_TOPICS
-	$(DOCKER) container inspect kafka-broker &> /dev/null || { echo "error:start kafka-broker container by 'make kafka-up'"; exit 1; }
+	$(DOCKER) container inspect $(KAFKA_CONTAINER_NAME) &> /dev/null || { echo "error:start kafka container by 'make kafka-up'"; exit 1; }
 	for topic in $(KAFKA_TOPICS); do \
-	    $(DOCKER) exec kafka-broker /opt/kafka/bin/kafka-topics.sh --create --topic $$topic --bootstrap-server localhost:9092; \
+	    $(DOCKER) exec $(KAFKA_CONTAINER_NAME) /opt/kafka/bin/kafka-topics.sh --create --topic $$topic --bootstrap-server localhost:9092; \
 	done
 
 .PHONY: kafka-topics-describe
 kafka-topics-describe:  ## Execute kafka-topics.sh for KAFKA_TOPICS
-	$(DOCKER) container inspect kafka-broker &> /dev/null || { echo "error:start kafka-broker container by 'make kafka-up'"; exit 1; }
+	$(DOCKER) container inspect $(KAFKA_CONTAINER_NAME) &> /dev/null || { echo "error:start kafka container by 'make kafka-up'"; exit 1; }
```

I suggest to reference `make compose-up`:

```suggestion
	$(DOCKER) container inspect $(KAFKA_CONTAINER_NAME) &> /dev/null || { echo "error:start kafka container by 'make compose-up'"; exit 1; }
```